raiden/tests/scenarios/ms2_simple_monitoring.yaml

```diff
@@ -72,6 +72,6 @@ scenario:
       - start_node: 1
 
       ## Monitored channel must be settled before the monitoring service can claim its reward
-      ## To make sure the transactions gets mined in time, 10 additional blocks are added
-      - wait_blocks: 5
+      ## To make sure the transactions gets mined in time, 20 additional blocks are added
+      - wait_blocks: 10
```

Comment:
comment and wait don't match

Reply:
True. Sorry, I was distracted. Will give it a thorough look